src/filter.ts

```diff
@@ -108,12 +109,12 @@ export function isOneOfFilter(filter: any): filter is OneOfFilter {
  * Converts a filter into an expression.
  */
 // model is only used for selection filters.
-export function expression(model: Model, filterOp: LogicalOperand<Filter>): string {
+export function expression(model: Model, filterOp: LogicalOperand<Filter>, node?: DataFlowNode): string {
   return logicalExpr(filterOp, (filter: Filter) => {
     if (isString(filter)) {
       return filter;
     } else if (isSelectionFilter(filter)) {
-      return predicate(model, filter.selection);
+      return predicate(model, filter.selection, node);
     } else { // Filter Object
       const fieldExpr = filter.timeUnit ?
         // For timeUnit, cast into integer with time() so we can use ===, inrange, indexOf to compare values directly.
```

Comment:
No, I'm not sure what these todos are referencing exactly. This PR is focused on how selections applied across other views depend on time units.